src/index.html

```diff
@@ -73,6 +73,74 @@ <h2 class="error-title">Error Loading Data</h2>
       <div class="bottom-zigzag">
         <img src="media/zigzag-pattern-bottom.svg" alt="Zigzag Pattern" class="zigzag" />
       </div>
+
+      <!-- QA Section -->
+      <div class="container">
+        <div class="qa-container">
+          <div class="qa-item">
+            <div class="question">How are networks ranked?</div>
+            <div class="answer">
+              <p>
+                Networks are ranked based on their retrieval score: the higher the score, the higher
+                the position on the leaderboard.
+              </p>
+            </div>
+          </div>
+
+          <div class="qa-item">
+            <div class="question">How is the retrieval score determined for Filecoin?</div>
+            <div class="answer">
+              <p>
+                The Filecoin retrieval score represents the average of all Storage Provider (SP)
+                retrieval success rate (RSR) scores over a 24-hour period. Storage Provider RSR is
+                calculated by dividing the number of successful retrievals by the total number of
+                valid retrieval requests for that provider within a given timeframe. Only requests
+                that pass fraud detection are included in the calculation. For more detail please
+                refer to
+                <a href="https://docs.filspark.com/" class="link">Spark protocol docs</a>.
+              </p>
+            </div>
+          </div>
+
+          <div class="qa-item">
+            <div class="question">How is the retrieval score determined for Arweave?</div>
+            <div class="answer">
+              <p>
+                The retrieval score for Arweave is determined by the percentage of successful
+                retrievals over a 24-hour period. Retrievals are conducted by selecting a random
+                Arweave transaction and performing a retrieval from a random Arweave node, and the
+                results are reported back to the API (without fraud checking). For more details,
```

Comment:
```suggestion
                results are reported back to the API. For more details,
```

No need to be vocal about this limitation